LibertyDSNP/sdk-ts
# update to node 16

Problem
=======
Updating Nodejs from 14 to 16
#183 

Solution
========
I updated the proper config and tool versions from node engine from 14.xx.xx to 16.14.0

Double Checks:
---------------
- [] Any new modules need to be exported?
- [] Are new methods in the right module?
- [] Are new methods/modules in core or not (i.e. porcelain or plumbing)?
- [] Do you have good documentation on exported methods?

Change summary:
---------------
* Changed the asdf install to node 16
* Updated tool-version, .node-version, github actions, package json
* Removed 'npm update v7' github action

Steps to Verify:
----------------
1. run: asdf install node 16
2. run: asdf which node (ensure the install is node 16.14.0
3. sdk-ts: npm install
4. sdk-ts: npm run build
5. sdk-ts: npm link
6. go to example-client
7. npm link @LibertyDSNP/sdk
8. compile and run example-client
9. sdk-ts: npm run watch
